components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/openidconnect/dao/RequestObjectDAOImpl.java

```diff
@@ -76,8 +77,8 @@ public void insertRequestObjectData(String consumerKey, String sessionDataKey, L
             String dbProductName = connection.getMetaData().getDatabaseProductName();
             prepStmt = connection.prepareStatement(sqlStmt, new String[]{
                     DBUtils.getConvertedAutoGeneratedColumnName(dbProductName, ID)});
-            prepStmt.setString(1, consumerKey);
```

Comment:
Is there a reason to remove the consumerkey from here?

Reply:
No. Seems like a mistake. Fixed it in https://github.com/wso2-extensions/identity-inbound-auth-oauth/pull/2423/commits/7cb82a605dd69fb710d6b4ff97a6c057159a8efb

Reply:
Fixed from https://github.com/wso2-extensions/identity-inbound-auth-oauth/pull/2423/commits/7cb82a605dd69fb710d6b4ff97a6c057159a8efb